src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -127,7 +127,7 @@ public function send(string $body, array $headers, int $delay = 0): string
             $now,
             $availableAt,
         ], [
-            null,
+            Type::BLOB,
```

Comment:
This is the right thing to do. But someone needs to test if it keeps working for people that still have the text type. So can you insert non-binary data to the body column if it is of type text (before this change)? Otherwise this PR would be a bc break.